src/common-components/Logistration.jsx

```diff
@@ -12,19 +16,47 @@ import { RegistrationPage } from '../register';
 
 const Logistration = (props) => {
   const { intl, selectedPage } = props;
+  const [institutionLogin, setInstitutionLogin] = useState(false);
+
+  const handleInstitutionLogin = () => {
+    if (selectedPage === LOGIN_PAGE) {
+      sendTrackEvent('edx.bi.institution_login_form.toggled', { category: 'user-engagement' });
+      sendPageEvent('login_and_registration', 'institution_login');
```

Comment:
And when the user clicks the back button the page event name should be `login` or `register` depending upon where it came from.

Reply:
I moved this login from the Login component to Logistration but this event tracking was only enabled for Login

Reply:
I guess there might be some misunderstanding while implemented in the past, you can verify the expected functionality here https://edge.edx.org/register using the `Use my institution/campus credentials` button.

Reply:
Updated the tracking event